### DISCUSSION TITLE
Resuming Subscriptions

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#563] [Help] [answered]
I'm creating a custom portal...

Is there a specific API call or method designed for resuming subscriptions?

I keep getting this error: Error updating subscription: Error: You are not allowed to update `status`

Thanks,
Shawn

**Comment by logeshswell** [comment, marked as answer]
Can you try updating `paused: false`?
```json
{
  "paused": false,
  "date_pause_end": null
}
```
